packages/lib/src/spatial-navigation/components/ScrollView.tsx

```diff
@@ -59,9 +152,80 @@ export const SpatialNavigationScrollView = ({
         showsHorizontalScrollIndicator={false}
         showsVerticalScrollIndicator={false}
         scrollEnabled={false}
+        onScroll={(event) => {
+          scrollY.current = event.nativeEvent.contentOffset.y;
+        }}
+        scrollEventThrottle={16}
       >
         {children}
       </ScrollView>
+      {deviceType === 'remotePointer' ? (
+        <PointerScrollArrows
+          topArrow={topArrow}
+          bottomArrow={bottomArrow}
+          topArrowContainerStyle={topArrowContainerStyle}
+          bottomArrowContainerStyle={bottomArrowContainerStyle}
+          onMouseEnterTop={onMouseEnterTop}
+          onMouseEnterBottom={onMouseEnterBottom}
+          webProps={webProps}
```

Comment:
isn't it weird to have webProps for something that is purely web?

Reply:
Maybe you can simply have two named props for webProps (not a spread object, but onMouseEnter etc.)

Reply:
Oh, and webProps is a function... this is very weird to name it this way 🤔

Reply:
I did a bunch of renaming and refactoring, should be ok now, and much clearer in my opinion :)
Looking forward to yours !